core/src/main/java/com/datastax/oss/driver/api/core/metrics/CoreNodeMetric.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.api.core.metrics;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Meter;
+
+public enum CoreNodeMetric implements NodeMetric {
+
+  /**
+   * The number of connections open to this node for regular requests (exposed as a {@link Gauge
+   * Gauge&lt;Integer&gt;}.
+   *
+   * <p>This does not include the control connection (which uses at most one extra connection to a
+   * random node in the cluster).
+   */
+  pooled_connection_count,
```

Comment:
I deliberately broke the convention because metric-land seems to favor lower case (from what I saw by looking at a few monitoring tools), and I find it nice to have a direct correspondence between the metric name and constant name.

If we want to respect the convention at all cost I could add a `getMetricName` that returns the lower-case version, but then it's a bit of a pity to have to explain that sometimes it's upper-case and sometimes lower.